pkg/extensions/search/common/common_test.go

```diff
@@ -55,9 +56,9 @@ var (
 	subRootDir string
 )
 
-type ImgResponsWithLatestTag struct {
-	ImgListWithLatestTag ImgListWithLatestTag `json:"data"`
-	Errors               []ErrorGQL           `json:"errors"`
+type RepoResponseWithNewestImage struct {
```

Comment:
Call this `RepoWithNewestImageResponse` instead

Reply:
Renamed